src/features/content-answers/ContentAnswersModalContent.tsx

```diff
@@ -1,38 +1,77 @@
 import React from 'react';
 
-import Answer from './Answer';
-import Question from './Question';
-import WelcomeMessage from './WelcomeMessage';
-
-import { QuestionType } from './ContentAnswersModal';
+import ContentAnswersModalChat from './ContentAnswersModalChat';
 // @ts-ignore: no ts definition
 // eslint-disable-next-line import/named
 import { User } from '../../../common/types/core';
 
-import './ContentAnswersModalContent.scss';
+export type QuestionType = {
+    prompt: string;
+    answer?: string;
+    createdAt?: string;
+};
 
 type Props = {
-    currentUser: User;
+    currentUser?: User;
```

Comment:
I believe currentUser will always be there, it's not optional?

Reply:
I think that could be a chance that the fetchCurrentUser function failed and maybe that is reason why the currentUser is set to optional. And I checked another usage in buie which also sets to optional.
https://github.com/box/box-ui-elements/blob/22bb03133fd84b13400757f78e00536a5461c648/src/elements/content-sidebar/Sidebar.js#L36

Reply:
when you initially fetch for the current user while youre loading the current user its not there yet thats why it has to be optional otherwise exceptions will throw because its trying to use it when still not available because the request hasnt returned yet